.github/workflows/node.js.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ dev, master ]
+  pull_request:
+    branches: [ dev, master ]
+
+jobs:
+  build:
+    services:
+      mongodb:
+        image: mongo
+        ports:
+          - 27017:27017
+    environment: testing
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
This doesn't match the node version specified in `.nvmrc` If moving to 14.x is intentional, `.nvmrc` should also be moved.